# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../xmms-jack/jack_free-function-nameclash-fix.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Avoid jack_free() function nameclash.

jack_free() is a public function provided by the jack API!

--- xmms-jack/jack.c.orig	2010-08-03 15:07:48.960091426 +0200
+++ xmms-jack/jack.c	2010-08-03 15:09:05.972178851 +0200
@@ -288,7 +288,7 @@
 
 
 /* Return the amount of data that can be written to the device */
-gint jack_free(void)
+gint xmms_jack_free(void)
 {
   unsigned long return_val = JACK_GetBytesFreeSpace(driver);
   unsigned long tmp;
@@ -639,7 +639,7 @@
 	jack_close,
 	jack_flush,
 	jack_pause,
-	jack_free,
+	xmms_jack_free,
 	jack_playing,
 	jack_get_output_time,
 	jack_get_written_time,
